qiskit/algorithms/minimum_eigen_solvers/vqe.py

```diff
@@ -577,7 +525,12 @@ def compute_minimum_eigenvalue(
         self._ret = result
 
         if aux_operators is not None:
-            aux_values = self._eval_aux_ops(opt_result.x, aux_operators, expectation=expectation)
+            param_dict = dict(zip(self.ansatz.ordered_parameters, opt_result.x))
+            bound_ansatz = self.ansatz.bind_parameters(param_dict)
```

Comment:
`ordered_parameters` is only an attribute for `BlueprintCircuit`s, not for any `QuantumCircuit`, so we can't use that here. But we don't have to pass dictionaries at all since a while and can just pass the final parameter vector 🙂 
```suggestion
            bound_ansatz = self.ansatz.bind_parameters(result.optimal_point)
```